Launcher/Storyboards/Base.lproj/Main.storyboard

```diff
@@ -1152,22 +1152,11 @@
             <objects>
                 <viewController title="Calabash Launcher Setup" storyboardIdentifier="settingswindow" id="Nd7-tF-bGG" customClass="SettingsViewController" customModule="Calabash_Launcher" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" id="0iB-dR-do5">
-                        <rect key="frame" x="0.0" y="0.0" width="878" height="376"/>
+                        <rect key="frame" x="0.0" y="0.0" width="878" height="406"/>
                         <autoresizingMask key="autoresizingMask"/>
                         <subviews>
-                            <button verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="jPd-p7-nqK">
-                                <rect key="frame" x="556" y="17" width="136" height="28"/>
-                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
-                                <buttonCell key="cell" type="push" title="Close" bezelStyle="rounded" alignment="center" controlSize="small" borderStyle="border" imageScaling="proportionallyDown" inset="2" id="It8-AQ-Odb">
-                                    <behavior key="behavior" pushIn="YES" lightByBackground="YES" lightByGray="YES"/>
-                                    <font key="font" metaFont="smallSystem"/>
-                                </buttonCell>
-                                <connections>
-                                    <action selector="clickCloseButton:" target="Nd7-tF-bGG" id="Tkb-Zm-wTR"/>
-                                </connections>
```

Comment:
I am on my mobile device and can’t see the close button being added anywhere - can you help me find it? 😅